Library/ViewModels/PledgePaymentPlansOptionViewModelTest.swift

```diff
@@ -135,6 +152,25 @@ final class PledgePaymentPlansOptionViewModelTest: TestCase {
     self.vm.inputs.optionTapped()
     self.notifyDelegatePaymentPlanOptionSelected.assertValue(.pledgeOverTime)
   }
+
+  func testPaymentPlanOption_PledgeOverTime_Ineligible() {
+    let project = Project.template
+    let ineligibleText = "Available for pledges over $150"
```

Comment:
let's add the TODO comment that you've been adding to this line as well!
`// TODO: add strings translations [MBL-1860](https://kickstarter.atlassian.net/browse/MBL-1860)`